### ISSUE TITLE
Scaled Log Sum Exp Atom

### ISSUE THREAD

**Issue by dcajasn** [cvxgrp/cvxpy#1203]
**Is your feature request related to a problem? Please describe.**
I want to model Mean - Entropic Value at Risk (EVaR) portfolio optimization models, following the paper __[Portfolio optimization with entropic value-at-risk](https://arxiv.org/pdf/1708.05713.pdf)__. I would like to express the problem as below in pseudo code:

``` python
w = cp.Variable((m, 1))  # Assets weights, 'm' assets
y = cp.Variable((1, 1), nonneg=True)  
n = 1000 # Number of observations
alpha = 0.05 # Confidence level of EVaR
risk = 1/y * cp.log_sum_exp(y * - returns @ w) - 1/y * cp.log (n * alpha)
objective = cp.Minimize(risk)
constraints = [cp.sum(w)==1, w>=0]
problem = cp.Problem(objective, constraints)
problem.solve()
``` 

The problem is that cvxpy doesn't allow to multiply variables except in two atoms: quad_over_lin(X, y) and kl_div(x,y). However in the paper it is shown that the objective function (EVaR) is convex for y > 0.

**Describe the solution you'd like**
It would be nice that the new atom log_sum_exp_y(X, y) will be implemented. The new atom is a scaled log_sum_exp, in pseudo code: 

``` python
cp.log_sum_exp_y(X, y) = 1/y * cp.log_sum_exp(y * X) # y > 0
``` 

**Describe alternatives you've considered**
I tried to write the atom based on the atoms `quad_over_lin`, `kl_div` and `log_sum_exp`; however, I always get an error.

**Additional context**
I am developing a portfolio optimization library called [Riskfolio-Lib](https://github.com/dcajasn/Riskfolio-Lib) based on cvxpy and It would be nice, if is posible, to model the EVaR with cvxpy because it would make easy the interaction with other risk measures and constraints.

**Comment by aszekMosek** [comment]
This: ``y * - returns @ w`` is hard to parse, even in pseudocode :). What is missing?

You can do 

```
t >= 1/y * log( exp(a_1*y) + exp(a_2*y) + ... + exp(a_n*y) )
```
by using the strategy with perspective of log-sum-exp https://docs.mosek.com/modeling-cookbook/practical.html#perspective-functions combined with ``s >= 1/y``.

By the way the ``- 1/y * cp.log (n * alpha)`` part is not DCP since the log is positive.

All in all my best guess is that you would be better off having a variable for ``1/y`` instead of ``y``.

**Comment by dcajasn** [comment]
Hi @aszekMosek, thanks for your answer. I'm going to make the change of variable `y` instead of `1/y` (I wrote the issue using `1/y` following the paper notation but I also tried with 'y'). I don't know that the scaled log_sum_exp function is called perspective function and can be modeled using that approach, thank you very much for share your knowledge 😃. I'm going to try it. If that doesn't work I will asume a normal distribution like this paper [Distributionally Robust Discrete Optimization with Entropic Value-at-Risk](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.1082.4511&rep=rep1&type=pdf) and see what are the advantages or disadvantages of this formulation to see if I included in my library.
